src/wp-admin/includes/user.php

```diff
@@ -145,7 +159,7 @@ function edit_user( $user_id = 0 ) {
 	}
 
 	// Check for "\" in password.
-	if ( false !== strpos( wp_unslash( $pass1 ), "\\" ) ) {
+	if ( false !== strpos( wp_unslash( $pass1 ), '\\' ) ) {
```

Comment:
The double quotes are needed, or change the literal.

Reply:
Seems to match upstream so I suspect it is literal.
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/user.php#L171

Reply:
Both are wrong, then, because it doesn't match the comment when using single quotes.

Reply:
I think I was misreading it. My mistake.